### ISSUE TITLE
Default tiers

### ISSUE THREAD

**Issue by xdamman** [opencollective/opencollective#2233]
Related to #2213 
We need good default tiers.

I'd argue that it's probably something that the host would want to change. So I'd like to propose default tiers for everyone and another set of default tiers just for open source collectives.

To try to improve our process, I'd love to set from the get go who will be the ultimate decision maker.

I think that for the Open Source Collectives, it should be @alanna and/or @piamancini.
And for the generic ones it should be me. Does that sound ok?

Then, we need to agree on the context. Who is the main audience for each category (generic and open source) and what are they trying to do independently of what we are trying them to do?

Here is a first stab at it:

For generic collectives:
- Main audience are individuals. Very rarely companies
- As a visitor, I want to have a quick way to sign up to stay up to date with their activities or make a simple donation.
- As a member of the collective, I want to quickly register as a member and see where everybody else is ("I recognize those other members, I need to be there too")

Proposal
![](https://d.pr/free/i/rNz9Tw+)

Most collectives I talked to don't have a "sponsorship" plan.
They only want to accept donations and also ask their "members" to pay a "membership fee".
The button reflects the call to action. While technically each of those buttons mean "become a financial contributor", I think it's more adapted to use "donate" and "become a member".

Open source collectives:
- Main audience are companies and individuals who want to support the project
- As an individual, I want to make a quick donation or join the collective as a "backer"
- As an organization, I want to become a sponsor. I want my logo to show up on their collective page and hopefully also on their website / github.
- As an organization, I might be interested in the services the collective has to offer 

Proposal:
![](https://d.pr/free/i/G48Hjx+)

This will probably be a bit more controversial.
I kept the "financial contribution" as a type of tier except for "support" which is of type "service". I've just added a small icon to reflect when it's recurring. Note that the final design will also include the starting price for the tier and the frequency. So I don't think that people will get confused about it.

The goal of each tier is to define a different way to contribute that will appeal to different audiences / needs:
1- Just donate, one time or recurring. No other expectation
2- Become a (recurring) member / backer as an individual and get your name to show up
3- Become a (recurring) sponsor as an organization and get your logo to show up

In all cases, all of this is still under the big title "Contribute" so we still "double down on owning the term Contribute". And each tier emphasizes that it's a financial contribution.
We could even start the description of each tier by saying: "Contribute by..."

This is just a first proposal. Still need to sleep on it and get different perspectives. So don't hesitate.

**Comment by xdamman** [comment]
This query returns the list of tiers that have been modified by one of the admins

```sql
SELECT
  count(t.id) as "number of tiers", name,
  count(o.id) as "number of orders", 
  sum(o."totalAmount") / 100 as "total orders amount"
FROM "Tiers" t
LEFT JOIN "Orders" o ON o."TierId" = t.id
WHERE t."deletedAt" IS NULL
  AND t.type != 'TICKET'
  AND t."updatedAt" != t."createdAt"
  AND (o.currency = 'EUR' OR o.currency='USD')
GROUP BY name
ORDER BY "number of tiers" DESC
```

I've exported the results in the [open collective public drive](https://drive.opencollective.com) under Strategy ([TiersBreakdown.csv](https://drive.google.com/open?id=1tItJECePGhy7ZYb0-kL_guYyb9sn7JIb)).

**Comment by alanna** [comment]
I don't think we need to complicate this so much or separate out open source Collective defaults from other Collectives. First of all, we don't have a good way to track which Collectives are open source projects - many of them are in hosts other than the Open Source Collective. 

Secondly, all Collectives have their own context. For example, a conference will have a bunch of levels of one-off sponsorship, plus tickets. I don't think it's accurate to divide all Collectives into open source and 'other', and say all 'other' Collectives are focused on a 'membership' concept.

I don't think we should have any defaults using the word "membership", as it means something different to every Collective. 

> As a member of the collective, I want to quickly register as a member and see where everybody else is

What does "member" mean in this case? Do you mean a follower? Financial contributor? Core Contributor? This is why I don't like that word and why we agreed not to use it.

Looking at the designs above, I think we're repeating words too much and over designing this. We don't need to say " Contribute" "Become a financial contributor" "financial contribution" all in a row before we even get to the tier title. 

If we remove the text title "Contribute" and the "financial contribution" blue label thing, which both seem redundant, then we can get away with the section title being "Become a financial contributor" and the default tiers being focused on the word contribute, which is what we want.

* Contribute Monthly
* Contribute Yearly
* Custom Contribution

Done!

This also removes the word "donate" which I think further confuses things. We want to keep this simple and to the point.

I very much want Collectives to get in there and customize their tiers. That's when we'll see words like "donate" "membership" and "sponsor". In fact I think the more boring the defaults, the more motivated core contributors will be to customize, which is the best outcome. Core contributors know their communities way better than we ever could.

For defaults, we should keep it super simple and generic, because we are trying to be a platform for all kinds of Collectives, and if we try to define any words beyond "contribute" which we decided to focus on, it will quickly stray into confusing territory where we are using terms differently than others understand them.

If we are thinking as we said we wanted to, as a company building a generic platform for all kinds of communities, then really tier defaults should be defined at the host level. But until we get to that point, we should keep it as simple as possible.
